updater/fetch.go

```diff
@@ -3,15 +3,20 @@ package updater
 import (
 	"bytes"
 	"context"
+	"errors"
 	"fmt"
+	"hash"
 	"io"
+	"io/ioutil"
 	"net/http"
 	"net/url"
 	"os"
 	"path"
 	"path/filepath"
 	"time"
 
+	"github.com/safing/jess/filesig"
```

Comment:
We now have a circular dependency between jess and portbase ... so version management will be a pain from now on. I'd suggest removing the portbase dependency from jess and, if we really need dsd in jess move that to a separate package under the github safing org.